build.gradle.kts

```diff
@@ -118,12 +119,14 @@ configure(subprojects.filterNot { it in internalBomModules }) {
         testImplementation("org.springframework.boot:spring-boot-starter-test") {
             exclude(group = "org.junit.vintage", module = "junit-vintage-engine")
         }
-        testImplementation("io.mockk:mockk:1.12.5")
+        // Combined with Kotlin 1.7.10 mockk 1.12.5 fails to 'every' some classes
+        // * https://github.com/mockk/mockk/issues/868
+        testImplementation("io.mockk:mockk:1.12.4")
```

Comment:
Mocks failed in `DgsDataFetchingEnvironmentTest` due to a conflict between Kotlin `1.7.10` and mockk `1.12.5`. A mockk downgrade to `1.12.4` resolved the issue.

Waiting on fix to be released here: https://github.com/mockk/mockk/issues/868

Reply:
This appears to be fixed with the laster version on `main`.